src/phpdoc/type_parser.go

```diff
@@ -63,6 +63,12 @@ const (
 	// Args[1] - val expression (right)
 	ExprKeyVal
 
+	// ExprMemberType is access to member.
+	// Examples: `\Foo::SOME_CONSTANT` `\Foo::$a`
+	// Args[0] - class fqn expression (left)
```

Comment:
Do we really have a guarantee that "left" is always FQN? I think it can be, in theory, a local type name like `Foo` without a forward slash.